### DISCUSSION TITLE
[npm]Run code-server as service

### DISCUSSION THREAD

**Discussion by Drilmo** [coder/code-server#5735] [Q&A]
Hi,

i have installed code-server with npm on my raspberry, I don’t no how to configure the execStart section in the file : 

/etc/systemd/system/code-server.service

can you help me ?

**Comment by Drilmo** [comment]
I have used n to install node 16

**Reply by jsjoeio** [reply]
Hmm... for the `execStart` start, is it just `code-server` since you've installed it globally with `npm`?
